lib/bwa/messages/control_configuration.rb

```diff
@@ -7,6 +7,7 @@ class ControlConfiguration < Message
       attr_accessor :model, :version
 
       def initialize
+        super()
```

Comment:
don't use parens without arguments (unless to explicitly pass no arguments to a super call, when there _are_ arguments to this method call, which is not the case here)